app/electron/main.ts

```diff
@@ -295,13 +296,17 @@ function startElecron() {
     mainWindow.webContents.on('dom-ready', () => {
       const octokit = new Octokit();
 
-      async function fetchLatestRelease() {
-        const response = await octokit.request('GET /repos/{owner}/{repo}/releases/latest', {
+      async function fetchRelease() {
+        const githubReleaseURL = `GET /repos/{owner}/{repo}/releases/latest`;
+        const response = await octokit.request(githubReleaseURL, {
```

Comment:
This commit should probably be squashed with the one that introduces this code, because without it the original code is broken. So even though these are complex changes, the functionality needs to be there from the beginning or we may try a broken commit later.